contrib/nydusify/pkg/utils/constant.go

```diff
@@ -16,7 +16,7 @@ const (
 	LayerAnnotationNydusBlobSize      = "containerd.io/snapshot/nydus-blob-size"
 	LayerAnnotationNydusBlobIDs       = "containerd.io/snapshot/nydus-blob-ids"
 	LayerAnnotationNydusBootstrap     = "containerd.io/snapshot/nydus-bootstrap"
-	LayerAnnotationNydusFsVersion     = "containerd.io/snapshot/nydus-fs-version"
+	LayerAnnotationNydusRAFSVersion   = "containerd.io/snapshot/nydus-rafs-version"
```

Comment:
Perhaps it would be better to name it `fs-version`, `rafs` may not be understood by a stranger, we can then use `fs-version` consistently with the option provided by `nydus-image` in `buildkit`, `converter (in nydus snapshotter)` and `acceld`, let me rename it.